pkgs/development/tools/jq/default.nix

```diff
@@ -1,33 +1,35 @@
-{stdenv, fetchurl, oniguruma}:
-let
-  s = # Generated upstream information
-  rec {
-    baseName="jq";
-    version="1.5";
-    name="${baseName}-${version}";
-    url="https://github.com/stedolan/jq/releases/download/jq-1.5/jq-1.5.tar.gz";
-    sha256="0g29kyz4ykasdcrb0zmbrp2jqs9kv1wz9swx849i2d1ncknbzln4";
+{ stdenv, lib, fetchFromGitHub, autoreconfHook,  oniguruma }:
+
+stdenv.mkDerivation rec {
+  name = "jq-${version}";
+  version = "something";
+
+  # Fix for CVE-2015-8863 is in master,
+  # but no release has been made since then (latest release was last year)
```

Comment:
In this case it seems the 1.6 release is delayed and they are recommending to use master:
- https://github.com/stedolan/jq/issues/1243#issuecomment-249399788
- https://github.com/stedolan/jq/issues/1239#issuecomment-248303846

I also took a closer look at the patches and they are smaller than I thought, so I will investigate if backporting to 1.5 is feasible.